### PR TITLE
fix: event type has to match schema defined type e.g. FoodDrive

### PR DESCRIPTION
**Pull request by winniezheng**

### CODE REVIEW

File: src/schema/typeDefs/event/event.graphql [hunk 1]
```diff
@@ -28,5 +28,5 @@ type FoodDrive implements Event {
 }
 
 enum EventType {
-  FOODDRIVE
+  FoodDrive
```
Comment (chrisrabe) [inline src/schema/typeDefs/event/event.graphql:31]: I think we can leave this one. I put in all caps here

File: src/utils/constants.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 export const EventType = {
-  foodDrive: "FOODDRIVE",
+  foodDrive: "FoodDrive",
```
Comment (chrisrabe) [inline src/utils/constants.js:2]: We can live the value to all caps

**Review by chrisrabe — CHANGES_REQUESTED** [review]
Can leave the value of the foodType enum to all caps

File: src/utils/constants.js (hunk 2 above)
Comment (winniezheng) [reply src/utils/constants.js:2]: I ran into issues with querying - the type has to match the one defined in the schema which is "FoodDrive"

Comment (chrisrabe) [reply src/utils/constants.js:2]: Was the issue still there if the schema is in all caps too?

Comment (winniezheng) [reply src/utils/constants.js:2]: yeah has to match type FoodDrive in the event typedefs

**Review by chrisrabe — APPROVED** [review]

**Pull request merged**